src/components/Tabs/components/Tab/Tab.spec.js

```diff
@@ -14,12 +14,4 @@ describe('Tab', () => {
       expect(actual).toMatchSnapshot();
     });
   });
-
-  describe('accessibility', () => {
-    it('should meet accessibility guidelines', async () => {
-      const wrapper = renderToHtml(<Tab />);
-      const actual = await axe(wrapper);
-      expect(actual).toHaveNoViolations();
-    });
-  });
```

Comment:
👀

Reply:
There is no value in a11y testing single tab by itself. At least few tab roles should be present for e meaningful test, imo.